src/Extensions/SiteTreeSubsites.php

```diff
@@ -398,8 +398,9 @@ public static function contentcontrollerInit($controller)
             SSViewer::set_themes(ThemeResolver::singleton()->getThemeList($subsite));
         }
 
-        if ($subsite && i18n::getData()->validate($subsite->Language)
-            && !singleton(SiteTree::class)->hasExtension('TractorCow\Fluent\Extension\FluentSiteTreeExtension')) {
+        $ignore_subsite_language = Config::inst()->get(self::class, 'ignore_subsite_language');
+
+        if ($subsite && i18n::getData()->validate($subsite->Language) && !$ignore_subsite_language) {
```

Comment:
```suggestion
        if (!$ignore_subsite_language && $subsite && i18n::getData()->validate($subsite->Language)) {
```
No need to call `validate()` if we're ignoring the locale anyway.

Reply:
ok, i'll change this.